core/crypto/vrf/p256/p256_test.go

```diff
@@ -66,6 +74,60 @@ func TestH2(t *testing.T) {
 	}
 }
 
+func TestFromWrappedKey(t *testing.T) {
+	ctx := context.Background()
+	for _, tc := range []struct {
+		spec   *keyspb.Specification
+		keygen keys.ProtoGenerator
+	}{
+		{
+			spec: &keyspb.Specification{
+				Params: &keyspb.Specification_EcdsaParams{
+					EcdsaParams: &keyspb.Specification_ECDSA{
+						Curve: keyspb.Specification_ECDSA_P256,
+					},
+				},
+			},
+			keygen: func(ctx context.Context, spec *keyspb.Specification) (proto.Message, error) {
+				return der.NewProtoFromSpec(spec)
+			},
+		},
```

Comment:
I'm not convinced it's worth preemptively making something a tabular test when it only has one test case.

Reply:
I think the Go style guides strongly prefer table driven tests, even for one test case. 
It makes it much easier to add new tests.

Reply:
https://github.com/golang/go/wiki/TableDrivenTests uses the wording "If you ever find yourself using copy and paste when writing a test, think about [...] refactoring into a table-driven test". With only one test case, you're not eliminating any duplicated code - in fact you're adding more than necessary. If more test cases are required in the future, it's easy enough to refactor into table-driven tests then (as suggested by the wording above).

Reply:
do you mind if I leave the test as is?  I suspect we'll add tests in the future, and it's a bit more consistent with the other tests.  I acknowledge that it isn't 100% necessary in this case.

Reply:
Sure, if you have an idea of other test cases that'll be added then that makes sense. I was just pushing back on using table-driven tests by default for everything. In the future, I'd recommend only making it table-driven once it needs to be, potentially saving yourself the effort of addressing comments like the ones below.